templates/summit/call-for-collaboration.html

```diff
@@ -31,18 +31,11 @@
         <hr class="p-rule" />
         <div class="col">
           <h2>
-            We’ve got exciting things
+            Ubuntu Summit 25.10
             <br />
-            in the works for 2025.
+            October 23-24 Online
```

Comment:
IMO the copydoc implies 'Online' should be on a new line

Reply:
Let me know when you have a reply and I can re-review

Reply:
Thanks Pete, its ready for re-review. Moved "Online" on a new line.